src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -160,6 +163,37 @@ public static GapicContext parse(CodeGeneratorRequest request) {
             transport);
 
     Preconditions.checkState(!services.isEmpty(), "No services found to generate");
+
+    // Temporary workaround for Ads, who still need these resource names.
+    if (services.get(0).protoPakkage().startsWith("google.ads.googleads.v")) {
```

Comment:
Can you please put some TODO here (I hope we do plan to get rid of google.ads.googleads.v specific hack, don't we)?

Reply:
Will do. Up to you whether we should bring back all these "unused" resource name helper classes altogether, or just for Ads.